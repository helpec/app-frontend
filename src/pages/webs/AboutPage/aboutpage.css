.valores{
    display: flex;
    justify-content: flex-start;
    padding-top: 10px;
}

.valor{
    alignment-baseline: central;
    font-size: 18px;
    text-align: center;
    width: 100%;
}