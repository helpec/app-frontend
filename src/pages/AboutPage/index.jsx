import React, { Component, Fragment } from 'react';
import BasePage from '../BasePage';

import Autora from './autora.jpg';
import Logo from './logo.png';
import './aboutpage.css';

class AboutPage extends Component {
  
  render() {
    return (
      <Fragment>
        <BasePage title="Quem Somos">
            <section id="content">
                  <div className="container">
                  <div className="row">
                      <div className="span12">
                        <h5 className="introducao">Helpec - <strong>Help Epilepsia e Convulsão</strong></h5>
                        
                        <p className="apresentacao"> Muitos devem estar se perguntando o porque de termos escolhido essa causa para apoiar? </p>
                        <p className="apresentacao"> A resposta é simples pela falta ou dificuldade em encontrar informações quando precisei, já tive caso familiar e amigos que sofreram do assunto e não encontravam nada sobre o assunto.Além disso a falta de informação do assunto causa a criação de preconceitos e mitos sobre o assunto.</p>
                        <p className="apresentacao"> Helpec veio propor um mundo mais empático em que as pessoas possam compreendem a condição do próximo.</p>
                        <p className="apresentacao">Temos como objetivo a saúde dos pacientes de Epilepsia e Convulsão através da distribuição de conteúdos para a ajuda, prevenção e primeiros socorros aos pacientes, e suporte aos familiares.</p>
                        <p className="apresentacao">Acreditamos que a tecnologia é uma ferramenta capaz de revolucionar a saúde e uma ferramenta para conectar pessoas!</p>
                        <p className="apresentacao">Somos uma ferramenta para conectar pessoas. Conectamos os pacientes durante uma crise epilética e/ou convulsiva aos seus primeiros socorros (familiares e amigos) através do botão de pânico.</p>
                      </div>
                    </div>
                <div className="container">
                    <div className="row">
                      <div className="span12">
                        <div className="solidline">
                        </div>
                      </div>
                    </div>
                    </div>
          
                    <div className="container">
                    <div className="row">
                    <div className="span2 text-center">
                        <img src= {Logo} className="visao" alt="Missão"/>
                        <div className="valores">
                          <h6 className="valor">Missão</h6>
                        </div>
                      </div>
                      <div className="span10">
                        <div className="valores">
                          <p className="valores">Nossa missão e disseminar e esclarecer dúvidas, primeiros socorros, materiais para consulta voltados a ajudar dos portadores, familiares e amigos que sofrem com Epilepsia e Convulsão.</p>
                        </div>
                      </div>
                    </div>
                    </div>
                   
                    <div className="container">
                    <div className="row">
                    <div className="span2 text-center">
                        <img src= {Logo} className="visao" alt="Missão"/>
                        <div className="valores">
                          <h6 className="valor">Visão</h6>
                        </div>
                      </div>
                      <div className="span10">
                        <div className="valores">
                          <p className="valores">Como visão estamos propondo o uso da tecnologia a nosso favor com a criação e elaboração de um aplicativo mobile que a pessoa que estiver passando mal poderá enviar notificação aos contatos cadastrados ao acionar o botão de pânico.</p>
                        </div>
                      </div>
                    </div>
                    </div>
                    
                  <div className="container">
                    <div className="row">
                    <div className="span2 text-center">
                        <img src= {Logo} className="visao" alt="Missão"/>
                        <div className="valores">
                          <h6 className="valor">Valores</h6>
                        </div>
                      </div>
                      <div className="span10">
                        <div className="valores">
                          <p className="valores">Dentre nossos valores estão a respeito e diminuir o preconceito contra os portadores de epilepsia e convulsão oferecendo materiais informativos, dicas de primeiros socorros e o uso do aplicativo a fim de facilitar a vida das pessoas.</p>
                        </div>
                      </div>
                    </div>
                    </div>

                    <div className="container">
                    <div className="row">
                      <div className="span12">
                        <div className="solidline">
                        </div>
                      </div>
                    </div>
                    </div>
                    
                    <div className="container">
                    <div className="row">
                      <div className="span12">
                        <h4>Colaboradora</h4>
                      </div>
                      
                      <div className="span3">
                        <img src= {Autora} alt="Foto Camila Perfil"/>
                        <div className="roles">
                          <p className="lead">
                            <strong>Camila Lima</strong>
                          </p>
                        </div>
                      </div>
                      <div className="span9">
                        <div className="roles">
                          <p className="colaboradora">
                            Estudante em formação para desenvolvimento FullStack, buscando aprender e desenvolver oferecendo através da tecnologia maior facilidade e ajuda com diversos temas.
                          </p>
                        </div>
                      </div>
                    </div>
                    <div className="row">
                      <div className="span12">
                        <div className="solidline">
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
                
                </section>
        </BasePage>
      </Fragment>
    );
  }
}

export default AboutPage;