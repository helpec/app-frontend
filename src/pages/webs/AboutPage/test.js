import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from "react-router-dom";
import AboutPage from '.';

test('Testar pagina web AboutPage', () => {
    const component = renderer.create(
        <BrowserRouter>
            <AboutPage />
        </BrowserRouter>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
