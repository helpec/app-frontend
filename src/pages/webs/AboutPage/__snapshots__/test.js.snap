// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testar pagina web AboutPage 1`] = `
<div
  id="wrapper"
>
  <header>
    <div
      className="container "
    >
      <div
        className="row nomargin"
      >
        <div
          className="span12"
        >
          <div>
            <div
              className="headnav"
            >
              <ul>
                <li>
                  <a
                    href="/registro"
                    onClick={[Function]}
                  >
                    <i
                      className="icon-user"
                    />
                     Registrar
                  </a>
                </li>
                -                  
                <li>
                  <a
                    href="/login"
                    onClick={[Function]}
                  >
                    <i
                      className="icon-user"
                    />
                    Login
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="span4"
        >
          <div
            className="logo"
          >
            <img
              alt=""
              className="logo"
              src="/assets/img/logo.png"
            />
            <h1
              className="logo h1"
            >
              Helpec - Help Epilepsia e Convulsão 
            </h1>
          </div>
        </div>
        <div
          className="span8"
        >
          <div
            className="navbar navbar-static-top"
          >
            <div
              className="navigation"
            >
              <nav>
                <ul
                  className="nav topnav"
                >
                  <li>
                    <a
                      href="/"
                      onClick={[Function]}
                    >
                      <i
                        className="fa fa-cog"
                      />
                      Home
                    </a>
                  </li>
                  <li>
                    <a
                      href="/quemsomos"
                      onClick={[Function]}
                    >
                      <i
                        className="fa fa-cog"
                      />
                      Quem somos
                    </a>
                  </li>
                  <li>
                    <a
                      href="/falandosobre"
                      onClick={[Function]}
                    >
                      <i
                        className="fa fa-cog"
                      />
                      Falando sobre
                    </a>
                  </li>
                  <li>
                    <a
                      href="/socorros"
                      onClick={[Function]}
                    >
                      <i
                        className="fa fa-cog"
                      />
                      Primeiros Socorros
                    </a>
                  </li>
                  <li
                    className="dropdown"
                  >
                    <a
                      href="#"
                    >
                      Saiba Mais
                      <i
                        className="icon-angle-down"
                      />
                    </a>
                    <ul
                      className="dropdown-menu"
                    >
                      <li>
                        <a
                          href="/saibamais"
                          onClick={[Function]}
                        >
                          <i
                            className="fa fa-cog"
                          />
                          Artigos
                        </a>
                      </li>
                      <li>
                        <a
                          href="/mitos"
                          onClick={[Function]}
                        >
                          <i
                            className="fa fa-cog"
                          />
                          Mitos e Verdades
                        </a>
                      </li>
                    </ul>
                  </li>
                  <li>
                    <a
                      href="/contato"
                      onClick={[Function]}
                    >
                      <i
                        className="fa fa-cog"
                      />
                      Contato
                    </a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </div>
    </div>
  </header>
  <section
    id="inner-headline"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="span4"
        >
          <div
            className="inner-heading"
          >
            <h2>
              Quem Somos
            </h2>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    id="content"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="span12"
        >
          <h5
            className="introducao"
          >
            Helpec - 
            <strong>
              Help Epilepsia e Convulsão
            </strong>
          </h5>
          <p
            className="apresentacao"
          >
             Muitos devem estar se perguntando o porque de termos escolhido essa causa para apoiar? 
          </p>
          <p
            className="apresentacao"
          >
             A resposta é simples pela falta ou dificuldade em encontrar informações quando precisei, já tive caso familiar e amigos que sofreram do assunto e não encontravam nada sobre o assunto.Além disso a falta de informação do assunto causa a criação de preconceitos e mitos sobre o assunto.
          </p>
          <p
            className="apresentacao"
          >
             Helpec veio propor um mundo mais empático em que as pessoas possam compreendem a condição do próximo.
          </p>
          <p
            className="apresentacao"
          >
            Temos como objetivo a saúde dos pacientes de Epilepsia e Convulsão através da distribuição de conteúdos para a ajuda, prevenção e primeiros socorros aos pacientes, e suporte aos familiares.
          </p>
          <p
            className="apresentacao"
          >
            Acreditamos que a tecnologia é uma ferramenta capaz de revolucionar a saúde e uma ferramenta para conectar pessoas!
          </p>
          <p
            className="apresentacao"
          >
            Somos uma ferramenta para conectar pessoas. Conectamos os pacientes durante uma crise epilética e/ou convulsiva aos seus primeiros socorros (familiares e amigos) através do botão de pânico.
          </p>
        </div>
      </div>
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="span12"
          >
            <div
              className="solidline"
            />
          </div>
        </div>
      </div>
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="span2 text-center"
          >
            <img
              alt="Missão"
              className="visao"
              src={[Function]}
            />
            <div
              className="valores"
            >
              <h6
                className="valor"
              >
                Missão
              </h6>
            </div>
          </div>
          <div
            className="span10"
          >
            <div
              className="valores"
            >
              <p
                className="valores"
              >
                Nossa missão e disseminar e esclarecer dúvidas, primeiros socorros, materiais para consulta voltados a ajudar dos portadores, familiares e amigos que sofrem com Epilepsia e Convulsão.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="span2 text-center"
          >
            <img
              alt="Missão"
              className="visao"
              src={[Function]}
            />
            <div
              className="valores"
            >
              <h6
                className="valor"
              >
                Visão
              </h6>
            </div>
          </div>
          <div
            className="span10"
          >
            <div
              className="valores"
            >
              <p
                className="valores"
              >
                Como visão estamos propondo o uso da tecnologia a nosso favor com a criação e elaboração de um aplicativo mobile que a pessoa que estiver passando mal poderá enviar notificação aos contatos cadastrados ao acionar o botão de pânico.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="span2 text-center"
          >
            <img
              alt="Missão"
              className="visao"
              src={[Function]}
            />
            <div
              className="valores"
            >
              <h6
                className="valor"
              >
                Valores
              </h6>
            </div>
          </div>
          <div
            className="span10"
          >
            <div
              className="valores"
            >
              <p
                className="valores"
              >
                Dentre nossos valores estão a respeito e diminuir o preconceito contra os portadores de epilepsia e convulsão oferecendo materiais informativos, dicas de primeiros socorros e o uso do aplicativo a fim de facilitar a vida das pessoas.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="span12"
          >
            <div
              className="solidline"
            />
          </div>
        </div>
      </div>
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="span12"
          >
            <h4>
              Colaboradora
            </h4>
          </div>
          <div
            className="span3"
          >
            <img
              alt="Foto Camila Perfil"
              src={[Function]}
            />
            <div
              className="roles"
            >
              <p
                className="lead"
              >
                <strong>
                  Camila Lima
                </strong>
              </p>
            </div>
          </div>
          <div
            className="span9"
          >
            <div
              className="roles"
            >
              <p
                className="colaboradora"
              >
                Estudante em formação para desenvolvimento FullStack, buscando aprender e desenvolver oferecendo através da tecnologia maior facilidade e ajuda com diversos temas.
              </p>
            </div>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="span12"
          >
            <div
              className="solidline"
            />
          </div>
        </div>
      </div>
    </div>
  </section>
  <footer>
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="span4"
        >
          <div
            className="widget"
          >
            <h3
              className="widgetheading"
            >
              Nossas Páginas
            </h3>
            <ul
              className="link-list"
            >
              <li>
                <a
                  href="/quemsomos"
                  onClick={[Function]}
                >
                  Quem somos
                </a>
              </li>
              <li>
                <a
                  href="/falandosobre"
                  onClick={[Function]}
                >
                  Falando Sobre
                </a>
              </li>
              <li>
                <a
                  href="/socorros"
                  onClick={[Function]}
                >
                  Primeiro Socorros
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="span4"
        >
          <div
            className="widget"
          >
            <h3
              className="widgetheading"
            >
              Fale conosco
            </h3>
            <p>
              <i
                className="icon-phone"
              />
               (11) 98232-6391 
              <br />
              <i
                className="icon-envelope-alt"
              />
               camilam.oliveiralima@gmail.com
            </p>
          </div>
        </div>
        <div
          className="span4"
        >
          <div
            className="widget"
          >
            <h3
              className="widgetheading"
            >
              Redes Sociais
            </h3>
            <ul
              className="social-network"
            >
              <li>
                <a
                  data-placement="bottom"
                  href="https://www.facebook.com/"
                  title="Facebook"
                >
                  <i
                    className="icon-facebook icon-square"
                  />
                </a>
              </li>
              <li>
                <a
                  data-placement="bottom"
                  href="https://www.linkedin.com/in/camilamlima/"
                  title="Linkedin"
                >
                  <i
                    className="icon-linkedin icon-square"
                  />
                </a>
              </li>
              <span>
                &Camila Lima - Todos os direitos reservados.
              </span>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </footer>
  <button
    className="panico"
    onClick={[Function]}
    type="button"
  >
     Pânico
  </button>
</div>
`;
